lib/membrane_rtc_engine/engine.ex

```diff
@@ -144,24 +144,23 @@ defmodule Membrane.RTC.Engine do
   @type turn_server_t() :: ExLibnice.relay_info()
 
   @typedoc """
-  List of RTP extensions to use.
+  List of WebRTC extensions to use.
 
-  At this moment only `vad` extension is supported.
+  At this moment only VAD (RFC 6464) is supported.
```

Comment:
Why TWCC is not listed here?

Reply:
TWCC has its module in webrtc_plugin, but it's implementation in rtp_plugin is not merged yet and requires adjustment to the recent changes. I thought we should not mention it in RTC engine until it's ready to use.

Reply:
But in the end, this will be included in this PR, or it will be in another PR?

Reply:
My plan was not including TWCC in this PR and opening a new one once TWCC is available in rtp_plugin